CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 # main
 
-* Add support for responsive `float` style arguments.
+# 0.0.3
+
+* Add support for responsive `float` style argument.
 
     *Joel Hawksley*
 
+# 0.0.2
+
 * Add components:
     * Avatar
```

Comment:
pretty sure this was only merged after 0.0.2 was released.
Could you change it please?